aikau/src/main/resources/alfresco/pickers/SitePicker.js

```diff
@@ -0,0 +1,245 @@
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @module alfresco/pickers/SitePicker
+ * @extends module:alfresco/pickers/Picker
+ * @author Dave Draper
+ */
+define(["dojo/_base/declare",
+        "alfresco/pickers/Picker"],
+        function(declare, Picker) {
+
+   var sitesView = {
+      name: "alfresco/lists/views/AlfListView",
+      config: {
+         widgets: [
+            {
+               name: "alfresco/lists/views/layouts/Row",
+               config: {
+                  widgets: [
+                     {
+                        name: "alfresco/lists/views/layouts/Cell",
+                        config: {
+                           widgets: [
+                              {
+                                 name: "alfresco/renderers/Property",
+                                 config: {
+                                    propertyToRender: "title"
+                                 }
+                              }
+                           ]
+                        }
+                     },
+                     {
+                        name: "alfresco/lists/views/layouts/Cell",
+                        config: {
+                           width: "20px",
+                           widgets: [
+                              {
+                                 name: "alfresco/renderers/PublishAction",
+                                 config: {
+                                    publishPayloadType: "CURRENT_ITEM",
+                                    publishGlobal: false,
+                                    publishToParent: false
+                                 }
+                              }
+                           ]
+                        }
+                     }
+                  ]
+               }
+            }
+         ]
+      }
+   };
+
+   return declare([Picker], {
+
+      /**
+       * An array of the i18n files to use with this widget.
+       *
+       * @instance
+       * @type {Array}
+       */
+      i18nRequirements: [
+         {i18nFile: "./i18n/Picker.properties"},
+         {i18nFile: "./i18n/SitePicker.properties"}
+      ],
+
+      /**
+       * Whether to select multiple sites in the picker or just one
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       */
+      multiSite: false,
```

Comment:
Can we switch this to be "singleItemMode" to make it consistent with other pickers please